pom.xml

```diff
@@ -95,13 +95,6 @@
             </excludes>
             <reportNameSuffix>sponge_log</reportNameSuffix>
           </configuration>
-          <dependencies>
-            <dependency>
```

Comment:
I'm not sure where this dependency is used.

It turns out that this dependency prevents `maven-surefire-plugin` using Junit 5 Platform.

Reply:
It looks like this dependency was added while ago in https://github.com/googleapis/java-shared-config/pull/37 to fix an issue with `mvn verify` in java-speech.  To double check, perhaps we can run a test with java-speech to make sure that's still not the case?